XamlControlsGallery/.editorconfig

```diff
@@ -0,0 +1,109 @@
+﻿# EditorConfig  https://EditorConfig.org
+# Style as Code Minimal EditorConfig v.2020.02.04.0 https://styleascode.net
+
+# top-most EditorConfig file
+root = true
+
+# Uses spaces for indentation.
+[*]
+indent_style = space
```

Comment:
We should use tabs since they are more accessible than spaces for indentation.

Reply:
Compiler and runtime teams agree that 4 spaces should be used.

Reply:
Oh okay, didn't knew that. Sorry about my comment then.

Reply:
As mentioned, the rules in this config are only those already found both the EditorConfig [used by the runtime](https://github.com/dotnet/runtime/blob/master/.editorconfig), and that [used by the compiler](https://github.com/dotnet/roslyn/blob/master/.editorconfig), and that match exactly in how they are enforced.

None of these rules involve my own opinions.